CHANGELOG.md

```diff
@@ -3,3 +3,4 @@
 * Modifies `firebase ext:update` to not perform update if the extension is already up to date.
 * Print Firebase Console links for Extensions after operations.
 * Updated Firebase Extensions registry address.
+* Adds printing of available official extensions when `firebase ext:install` is run without an extension name.
```

Comment:
This seems to be burying the lede - the big feature here is the ability to choose and install an extension from the list! @rachelsaunders might be able to suggest better language, but maybe something like:

```Adds the ability to select an extension to install from the official list when `firebase ext:install` is run without an extension name.```

Reply:
What I have is actually Rachel's release note suggestion haha.

Reply:
I think that Joe is on the right improvement track for this release note.   His _actually_ captures what this PR does (it's not just that the CLI prints the list, it's that the dev can select from that list - woot!).  Just a small tweak to make it more clear:

Adds the ability to select an extension to install from a list of available official extensions when `firebase ext:install` is run without an extension name.

Reply:
Changed!